tutorials/tts/FastPitch_Finetuning.ipynb

```diff
@@ -326,7 +326,7 @@
                 "`\n",
```

Comment:
In general, it's better to use this whitelist: `https://github.com/NVIDIA/NeMo/blob/main/nemo_text_processing/text_normalization/en/data/whitelist/tts.tsv` , not needed for hi-fi tts as it's already normalized

Reply:
Should be a quick change. Would you prefer it in this PR or a new one?

Reply:
up to you. Some checks are still running - hmm, it's been 19+ hrs since the last commit